hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -148,6 +148,9 @@
     FRONTEND_STARTUP_TOPIC_METADATA_LOADING_RETRY_COUNT("frontend.startup.topic.metadata.loading.retry.count", 5),
     FRONTEND_STARTUP_TOPIC_METADATA_LOADING_THREAD_POOL_SIZE("frontend.startup.topic.metadata.loading.thread.pool.size", 16),
 
+    FRONTEND_TOPIC_METADATA_LOADING_JOB_ENABLED("frontend.startup.topic.metadata.job.enabled", false),
+    FRONTEND_TOPIC_METADATA_LOADING_JOB_INTERVAL("frontend.startup.topic.metadata.job.interval", 60),
```

Comment:
I think the `startup` word just doesn't fit well here, as it runs all the time. :)